packages/assets/scss/03-organisms/_mass-feedback-form.scss

```diff
@@ -58,10 +58,11 @@ $formQuickTransitions: max-height 0.3s linear;
 
 
     max-height: 0;
-    overflow: hidden;
+    display: none;
     transition: $formTransitions;
 
     &.is-open {
+      display: initial;
       max-height: 150vh;
```

Comment:
I think L. 66 max-height: 150vh; is responding to max-height: 0; and overflow: hidden; for div.ma__mass-feedback-form__fields. This line can be removed.

Reply:
Removed